docs/DeveloperGuide.adoc

```diff
@@ -28,12 +28,15 @@
 .. Open any test file in `\test\java` and place your cursor over any `@Test` highlighted in red
 .. Press ALT+ENTER and select `Add 'JUnit4' to classpath`
 .. Select `Use 'JUnit4' from IntelliJ IDEA distribution` and click `OK`
+. Add dependency on module `addressbook-level2`
+.. Open any test file in `\test\java` and place your cursor over any imports from `seedu.addressbook` highlighted in red
+.. Press ALT+ENTER and select `Add dependency on module 'addressbook-level2'`
```

Comment:
I wasn't able to get this option (`seedu.addressbook` doesn't highlight in red for me, yet execution of the test still fails anyway due to the missing module).

Reply:
You might want to try deleting the `.idea` folder, or re-clone the project, and try the instructions again. The existing configuration might be interfering with the how you import the modules.